Changelog.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
```

Comment:
Can you add this to the PR template?

Right now it says " If this PR marked a release, I updated CHANGELOG.md.", but I think updating the changelog should be mandatory every time. So maybe "I updated CHANGELOG.md." would work.

Reply:
Updated this in my new PR so ignore this comment.